person.rb

```diff
@@ -0,0 +1,29 @@
+require './nameable'
+
+class Person < Nameable
+  attr_accessor :name, :age, :rentals
+  attr_reader :id
+
+  def initialize(age, name = 'unknow', parent_permission = 'true')
+    super()
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def correct_name
+    @name
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  private
+
+  def of_age?
+    @gae >= 18
+  end
+end
```

Comment:
- [ ] Good job so far 👏 As a Person class has-many Rentals as a relationship/ association, you are supposed to create a method that adds rentals. As a hint, It should look like this 👇 
```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end

```
where `self` represents the Person object itself 😃 👍